userguide/support.rst

```diff
@@ -145,23 +142,17 @@ from a web browser.
 
 To get the most out of the IRC channel, keep these points in mind:
 
-* Do not ask "can anyone help me?". Just ask the question. If someone
-  knows the answer, they will try to help.
+* Do not ask "can anyone help me?". Just ask the question.
 
-* Do not ask a question and then leave. Users who know the answer
-  cannot help you if you disappear.
+* Do not ask a question and leave.
 
-* Do not take it personally if no one answers or demand that someone
-  answers your question. Maybe no one who knows the answer is
-  available, maybe your question is really difficult, or maybe it is a
-  question that has already been answered many times in the other
-  support resources. Try asking again in a few hours or research the
-  other resources to see if you have missed anything.
+* If no one answers, the question may be difficult to answer or it has
+  been asked before. Try asking again in a few hours or research other
```

Comment:
Rather than asking them to repost, we should suggest searching the forums or the web while they wait.